x11-wm/mutter/mutter-9999.ebuild

```diff
@@ -26,7 +26,7 @@ REQUIRED_USE="
 	gtk-doc? ( introspection )
 	wayland? ( ^^ ( elogind systemd ) udev )
 	xwayland? ( wayland )
-	test? ( wayland )"
+	test? ( screencast wayland )"
```

Comment:
This might be tweakable upstream to not run tests that can't run in the current configuration?